cyberbattle/__init__.py

```diff
@@ -91,3 +92,30 @@ def register(id: str, cyberbattle_env_identifiers: model.Identifiers, **kwargs):
             },
     reward_threshold=2200,
 )
+
+ad_envs = [f"ActiveDirectory-v{i}" for i in range(0, 10)]
+for env in ad_envs:
+    if env in registry.env_specs:
+        del registry.env_specs[env]
+
+    register(
+        id=env,
+        cyberbattle_env_identifiers=generate_ad.ENV_IDENTIFIERS,
```

Comment:
I dont' see where you pass the `i` parameter. Should not the size of the environment depend on `i`? Or to seed the random function? Otherwise they will all be identical.

Reply:
Actually, not identical, but still I wonder if we should initialize the random seed before each instantiation to make it reproducible.

Reply:
I added the ability to seed the environment with one of the kwargs